package.json

```diff
@@ -4,11 +4,13 @@
   "repository": "https://github.com/dimagi/commcare-hq",
   "description": "CommcareHQ Javascript tests",
   "dependencies": {
+    "biginteger": "^1.0.3",
```

Comment:
Ok, that works for me locally! Made this PR w/ version bump: https://github.com/dimagi/js-xpath/pull/11